README.md

```diff
@@ -14,6 +14,7 @@ This packages contains generic controllers that can be used for a biped, quadrup
 - BulletUtils (Optional, needed to run demos)
 - Robot_Properties_Solo (Optional, needed to run demos)
 - Robot_Properties_Bolt (Optional, needed to run demos)
+- Robot_Properties_Go1 (Optional, needed to run demos)
```

Comment:
we miss something in the CMakeLists.txt to check if those optional dependencies are installed or not...
It is not needed per say but would be nice IMHO.